_includes/download_buttons.html

```diff
@@ -1,10 +1,19 @@
-<div id="download">
+<div id="download" class="row">
+
   <p class="center">
-    Read The Book
-  </p>
-  <p class="center">
-    <a href="//book.costoffreedom.cc/"><button class="button-download button">HTML</button></a>
-    <a href="https://www.gitbook.com/download/pdf/book/costoffreedom/costoffreedom-book"><button class="button-download button">PDF</button></a>
-    <a href="https://www.gitbook.com/download/epub/book/costoffreedom/costoffreedom-book"><button class="button-download button">EPUB</button></a>
+    <span class="nine columns">
+      Read The Book
+      <br>
+      <br>
+      <a href="//book.costoffreedom.cc/"><button class="button-download button">HTML</button></a>
+      <a href="https://www.gitbook.com/download/pdf/book/costoffreedom/costoffreedom-book"><button class="button-download button">PDF</button></a>
```

Comment:
@clemsos should this point to https://github.com/costoffreedom/costoffreedom-book/raw/master/costoffreedom-final.pdf instead? Or maybe rather the file with a more canonical name (like say costoffreedom-edition-1.1.pdf on the website?) The LaTeX generated PDF looks a ton better, ideally nobody ever sees the gitbook generated one, right?